docs/error_codes.md

```diff
@@ -4,26 +4,31 @@ This document is suggesting that a high level error category 6xxx(can be another
 
 - **Permission Error** -- **61**_xx_
 
-| **Error Code** | **Name** | **Description** | /parties | /thirdPartyRequest | /consentRequests | /authorizations |
-| --- | --- | --- | --- | --- | --- | --- |
-| **6100** | Authentication rejection | Generic authentication rejection |  |  | X | X |
-| **6101** | Unsupported authentication channel | Authentication request is attempting to authorize with an authentication channel that the DFSP does not support. Example Web, OTP. |  |  |  | X |
-| **6102** | Unsupported scopes were requested | Authentication request is attempting to get authorization for scopes that the DFSP doesn’t allow/support |  | |  | X |
-| **6103** | Consent not given | DFSP denies user gave consent or DFSP says user has revoked consent. |  |  | X |  |
+| **Error Code** | **Name** | **Description** | /parties | /thirdPartyRequest | /consentRequests | /authorizations | /consents |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **6100** | Authentication rejection | Generic authentication rejection |  |  | X | X |  |
+| **6101** | Unsupported authentication channel | Authentication request is attempting to authorize with an authentication channel that the DFSP does not support. Example Web, OTP. |  |  |  | X |  |
+| **6102** | Unsupported scopes were requested | Authentication request is attempting to get authorization for scopes that the DFSP doesn’t allow/support |  | |  | X |  |
+| **6103** | Consent not given | DFSP denies user gave consent or DFSP says user has revoked consent. |  |  | X |  |  |
 
 - **Validation Error** -- **62**_xx_
 
-| **Error Code** | **Name** | **Description** | /parties | /thirdPartyRequest | /consentRequests | /authorizations |
-| --- | --- | --- | --- | --- | --- | --- |
-| **6200** | Validation error | Generic validation error. | X | X | X | X |
-| **6201** | Invalid signed challenge | PISP server receives signed challenge that does not match the original challenge. |  |  |  | X |
-| **6202** | Missing authentication credential | Payload received with missing authentication credential.  |  |  |  | X |
-| **6203** | Invalid authentication token | DFSP receives invalid authentication token from PISP.  |  |  |  | X |
-| **6204** | OTP is incorrect | One time password is incorrect.  |  |  |  |X|
+| **Error Code** | **Name** | **Description** | /parties | /thirdPartyRequest | /consentRequests | /authorizations | /consents |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| **6200** | Validation error | Generic validation error. | X | X | X | X | X |
+| **6201** | Malformed syntax | Generic validation error. | X | X | X | X | X |
+| **6202** | Missing mandatory element | The format of the parameter is not valid. For example, amount set to 5.ABC. The error description field should specify which element is erroneous | X | X | X | X | X |
+| **6203** | Too many elements | The number of elements of an array exceeds the maximum number allowed. |  |  |  |  | X |
```

Comment:
Shouldn't this be on all resources?

Reply:
It's specific to arrays. So it should only affect /parties `accounts` field and /consents and /consentRequests `scopes` field. As far as I know.
Thought I had updated this.
Marked /parties and /consentRequests as well.